lib/src/default_emoji_picker_view.dart

```diff
@@ -163,105 +215,77 @@ class _DefaultEmojiPickerViewState extends State<DefaultEmojiPickerView>
     return GestureDetector(
       onTap: _closeSkinToneDialog,
       child: GridView.count(
-        scrollDirection: Axis.vertical,
-        physics: const ScrollPhysics(),
-        controller: _scrollController,
-        shrinkWrap: true,
-        primary: false,
-        padding: widget.config.gridPadding,
-        crossAxisCount: widget.config.columns,
-        mainAxisSpacing: widget.config.verticalSpacing,
-        crossAxisSpacing: widget.config.horizontalSpacing,
-        children: categoryEmoji.emoji.asMap().entries.map((item) {
-          final index = item.key;
-          final emoji = item.value;
-          final onPressed = () {
-            _closeSkinToneDialog();
-            widget.state.onEmojiSelected(categoryEmoji.category, emoji);
-          };
-
-          final onLongPressed = () {
-            if (!emoji.hasSkinTone || !widget.config.enableSkinTones) {
-              _closeSkinToneDialog();
-              return;
-            }
-            _closeSkinToneDialog();
-            _openSkinToneDialog(emoji, emojiSize, categoryEmoji, index);
-          };
-
-          return _buildButtonWidget(
-            onPressed: onPressed,
-            onLongPressed: onLongPressed,
-            child: _buildEmoji(
-              emojiSize,
-              categoryEmoji,
-              emoji,
-              widget.config.enableSkinTones,
-            ),
-          );
-        }).toList(),
-      ),
+          scrollDirection: Axis.vertical,
+          controller: _scrollController,
+          primary: false,
+          padding: widget.config.gridPadding,
+          crossAxisCount: widget.config.columns,
+          mainAxisSpacing: widget.config.verticalSpacing,
+          crossAxisSpacing: widget.config.horizontalSpacing,
+          children: [
+            for (int i = 0; i < categoryEmoji.emoji.length; i++)
+              _buildEmojiCell(
+                  emojiSize, categoryEmoji, categoryEmoji.emoji[i], i)
+          ]),
     );
   }
 
   /// Build and display Emoji centered of its parent
   Widget _buildEmoji(
     double emojiSize,
-    CategoryEmoji categoryEmoji,
     Emoji emoji,
     bool showSkinToneIndicator,
   ) {
-    // FittedBox needed for display, font scale settings
-    return FittedBox(
```

Comment:
I couldn't find any issue on android by removing `FittedBox`, but when I wanted to check iOS I saw that your current implementation show's nothing 😅 (iPhone 13 Pro Simulator)
<img src="https://user-images.githubusercontent.com/11667234/181584575-c7d46f04-4efa-4d42-8d43-24831e910f2a.png" alt="drawing" width="300"/>

Reply:
This is absolutely crazy. I don't think it is related to FittedBox, though, since it doesn't show categories tabs either. My iOS setup is annoying to deal with and just setting TargetPlatform does not help to reproduce the issue. I'll have a closer look tomorrow.

Reply:
ok, this was rather stupid on my part. Forgot to check platform and was calling native compatibility check method on all platforms. Fixed now.